src/pages/events.mdx

```diff
@@ -14,10 +13,12 @@ https://enarx.dev/resources/tags/event
 ## Upcoming Events
  
 
-### (All Things Open) Unlocking the Cloud with Confidential Computing
+### Shmoocon 2023
 
-<ATO1 />
+**Authors**: Richard Zak, Dmitri Pal  
```

Comment:
This is not a blocking comment for this PR, however, in the future, it might make sense to use something like "Presenters/Participants" rather than Authors. As is does not read right. But it can be updated next round.